juju_verify/verifiers/ceph.py

```diff
@@ -291,30 +327,75 @@ def check_cluster_health(cls, *units: Unit) -> Result:
         return result
 
     @classmethod
-    def get_replication_number(cls, unit: Unit) -> Optional[int]:
-        """Get minimum replication number from ceph-mon unit.
+    def get_crush_rules(cls, unit: Unit) -> Dict[int, CrushRuleInfo]:
+        """Get all crush rules in Ceph cluster."""
+        verify_charm_unit("ceph-mon", unit)
+        # NOTE (rgildein): This should be replaced be feature action in ceph-mon.
```

Comment:
typo: "replaced by".

It would be helpful if we could file a bug for the required action and add the `LP#<BUGID>` in this comment.

Reply:
I planned to do it, after this changes will get full reviewed. 
I report [LP#1957458](https://bugs.launchpad.net/charm-ceph-mon/+bug/1957458).